client/src/components/Voting/VotingArea/VotingArea.js

```diff
@@ -1,14 +1,47 @@
 import styled from 'styled-components/macro'
+import { useEffect, useState } from 'react'
 import { IconContext } from 'react-icons'
-import IdkArea from './IdkArea/IdkArea'
+import axios from 'axios'
+import UnknownArea from './UnknownArea/UnknownArea'
 import RatingArea from './RatingArea/RatingArea'
+import ErrorCard from '../../Messages/ErrorCard/ErrorCard'
+
+const VotingArea = ({ onVoteClick, firstMovieTmdbId, secondMovieTmdbId }) => {
+  const [movies, setMovies] = useState([])
+  const [componentError, setComponentError] = useState('')
+
+  useEffect(() => {
+    const fetchMovies = async (firstTmdbId, secondTmdbId) => {
+      try {
+        const responseMovieOne = await axios.get(`/api/tmdb/${firstTmdbId}`)
+        const responseMovieTwo = await axios.get(`/api/tmdb/${secondTmdbId}`)
+
+        setMovies([responseMovieOne.data, responseMovieTwo.data])
+      } catch (error) {
+        console.error(error)
+        setComponentError({ message: error.message })
+      }
+    }
+
+    if (movies.length < 2) fetchMovies(firstMovieTmdbId, secondMovieTmdbId)
+  }, [movies, firstMovieTmdbId, secondMovieTmdbId])
+
+  if (componentError !== '') {
+    return <ErrorCard title="Error" message={componentError.message} />
+  }
+
+  if (movies.length < 2) {
+    return <AreaVoting>Loading</AreaVoting>
+  }
 
-const VotingArea = () => {
   return (
     <AreaVoting>
       <IconContext.Provider value={{ size: '48px' }}>
-        <IdkArea />
-        <RatingArea />
+        <UnknownArea
+          firstMovieTitle={movies[0].title}
+          secondMovieTitle={movies[1].title}
```

Comment:
Personally I think it would be nicer to work with an object and get the data via the object properties than using an array and going via the index. But that's just me 🙂